templates/ddns.html

```
@@ -0,0 +1,190 @@
+              <input type="checkbox" id="cloudflareEnabled">
+              <span class="toggle-slider"></span>
+            </label>
+          </div>
+
+          <div class="mb-4">
+            <div class="fw-semi" style="font-size: var(--text-md);">Cloudflare Setup Instructions</div>
+            <ol class="mt-2 text-muted" style="font-size: var(--text-sm); line-height: 1.6; padding-left: 1.5rem; list-style-type: decimal;">
+              <li style="padding-left: 0.25rem;">Login to <a href="https://dash.cloudflare.com/" target="_blank" rel="noopener">Cloudflare Dashboard</a> and select your top-level domain.</li>
+              <li style="padding-left: 0.25rem;">Create a new 'A' record pointing to your server (e.g. <code>server.mydomain.com</code>). Make sure it exists!</li>
+              <li style="padding-left: 0.25rem;">Locate your <strong>Zone Identifier</strong> on the Overview page of your domain (bottom right).</li>
+              <li style="padding-left: 0.25rem;">Go to <a href="https://dash.cloudflare.com/profile/api-tokens" target="_blank" rel="noopener">My Profile -> API Tokens</a>.</li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)